data/interfaces/default/header.html

```diff
@@ -8,8 +8,8 @@
   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <link rel="stylesheet" type="text/css" href="interfaces/default/css/materialize.min.css">
```

Comment:
This should point to the theme dir and not just the default one if we're correcting stuff (```href = interfaces/${interface}/css/materialize.min.css```)

Reply:
Done.